app/config/sculpin_site.yml

```diff
@@ -16,6 +16,9 @@ subspecies:
     cakephp:
         common: "CakePHP"
         latin: "pistrini"
+    check24:
+        common: "CHECK24"
+        latin: "collatio XXIV"
```

Comment:
The proper form here would be `collationis`, although the number afterwards is weird. What if we drop the number? They don't have to be direct translations. A little subtly and humor is good.

Reply:
Done ;)